### PR TITLE
Add angle to halftone

### PR DESCRIPTION
**Pull request by TotoShampoin**
I saw the halftone node, and I immediately wanted to add an angle to it, so the user can do this

Screenshot_20231223_094341

### CODE REVIEW

File: Nodes/11 Image Modifier (Post Process)/Greyscale Halftone.clbnode.presets.json [hunk 1]
```diff
@@ -17,11 +43,11 @@
                                         "Min": 0.0,
                                         "Has max bound": false,
                                         "Max": 1.0,
-                                        "Drag speed": 0.10000000149011612,
+                                        "Drag speed": 0.009999999776482582,
```
Comment (JulesFouchy) [inline Nodes/11 Image Modifier (Post Process)/Greyscale Halftone.clbnode.presets.json:46]: I don't think this change was intentional, please discard it. Or if it is intentional, please explain why.

File: Nodes/11 Image Modifier (Post Process)/Greyscale Halftone.clbnode.presets.json [hunk 2]
```diff
@@ -17,11 +43,11 @@
...
                                         "Use slider": false
                                     }
                                 },
-                                "Default Value": 500.0,
+                                "Default Value": 0.0,
```
Comment (JulesFouchy) [inline Nodes/11 Image Modifier (Post Process)/Greyscale Halftone.clbnode.presets.json:50]: I don't think this change was intentional, please discard it. Or if it is intentional, please explain why.

**Comment by JulesFouchy** [comment]
This is a very nice effect indeed.

**Review by JulesFouchy — CHANGES_REQUESTED** [review]

File: Nodes/11 Image Modifier (Post Process)/Greyscale Halftone.clbnode.presets.json (hunk 2 above)
Comment (TotoShampoin) [reply Nodes/11 Image Modifier (Post Process)/Greyscale Halftone.clbnode.presets.json:50]: Both unintentional, yes. What is it supposed to be anyway?

Comment (JulesFouchy) [reply Nodes/11 Image Modifier (Post Process)/Greyscale Halftone.clbnode.presets.json:50]: Some default values for other parameters that were already there before your changes.

**Review by TotoShampoin — COMMENTED** [review]
Changed as you asked

**Pull request merged**